next/components/forms/widget-wrappers/WidgetWrapper.tsx

```diff
@@ -30,6 +36,26 @@ const WidgetWrapper = ({
       })}
     >
       {children}
+      {Array.isArray(accordion) &&
+        accordion.map((item) => {
+          return (
+            <Accordion
+              key={uuidv4()}
```

Comment:
instead of this use this approach (version for mapped elements) - https://bratislava.github.io/frontend/components#use-generated-ids